internal/view/login.html

```diff
@@ -109,6 +109,7 @@
 
 						// Save account data
 						localStorage.setItem("shiori-account", JSON.stringify(json.account));
+                        localStorage.setItem("shiori-setting", JSON.stringify(json.account.config));
```

Comment:
Why are you not getting the config from the `shiori-account` storage? It's getting stored there as well.

Reply:
And why not update the settings stored in `shiori-account` when the user modifies them?